--- snap-core-0.9.5.0/snap-core.cabal.orig	2013-11-05 20:04:15.000000000 +0100
+++ snap-core-0.9.5.0/snap-core.cabal	2013-12-26 14:02:03.811250235 +0100
@@ -147,7 +147,7 @@
     mtl                       >= 2.0     && < 2.2,
     random                    >= 1       && < 2,
     regex-posix               >= 0.95    && < 1,
-    text                      >= 0.11    && < 0.12,
+    text                      >= 0.11    && < 1.1,
     time                      >= 1.0     && < 1.5,
     unix-compat               >= 0.2     && < 0.5,
     unordered-containers      >= 0.1.4.3 && < 0.3,
